src/main/kotlin/org/jitsi/jibri/service/impl/SipGatewayJibriService.kt

```diff
@@ -85,14 +88,24 @@ class SipGatewayJibriService(
     override fun start() {
         jibriSelenium.joinCall(
             sipGatewayServiceParams.callParams.callUrlInfo.copy(urlParams = SIP_GW_URL_OPTIONS))
-        whenever(jibriSelenium).transitionsTo(ComponentState.Running) {
+
+        // when in auto-answer mode we want to start as quick as possible as
+        // we will be waiting for a sip call to come
+        if (sipGatewayServiceParams.sipClientParams.autoAnswer) {
             pjsuaClient.start()
+        } else {
```

Comment:
so it's ok here if the pjsua client starts (and maybe connects?) before the browser has started/joined the call? won't we potentially forward the desktop background or something there?

Reply:
We will see, in the past, we were setting some branded background image so it doesn't look weird.

The point is that loading the web can take a while and we will be trying to send a sip call to be established and we expect port 5060 to be open.